src/analysis/locally_convex/bounded.lean

```diff
@@ -0,0 +1,109 @@
+/-
+Copyright (c) 2022 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll
+-/
+import analysis.normed.normed_field
+import analysis.locally_convex.basic
+import topology.algebra.module.basic
+import topology.bornology.basic
+
+/-!
+# Natural Boundedness
+
+This file defines natural or von Neumann bounded sets and proves elementary properties.
+
+## Main declarations
+
+* `is_bounded`: A set `s` is bounded if every neighborhood of zero absorbs `s`.
+* `natural_bornology`: The set of bounded sets forms a bornology.
+
+## Main results
+
+* `is_bounded_of_topological_space_le`: A coarser topology admits more bounded sets.
+
+## References
+
+* [Bourbaki, *Topological Vector Spaces*][bourbaki1987]
+
+-/
+
+variables {𝕜 E : Type*}
+
+open_locale topological_space pointwise
+
+namespace bornology
+
+section semi_normed_ring
+
+section has_zero
+
+variables (𝕜)
+variables [semi_normed_ring 𝕜] [has_scalar 𝕜 E] [has_zero E]
+variables [topological_space E]
+
+/-- A set `s` is von Neumann bounded if every neighborhood of 0 absorbs `s`. -/
+def is_vonN_bounded (s : set E) : Prop := ∀ V ∈ 𝓝 (0 : E), absorbs 𝕜 V s
+
+variables (E)
+
+@[simp] lemma is_vonN_bounded.empty : is_vonN_bounded 𝕜 (∅ : set E) :=
```

Comment:
I know @j-loreaux asked you to use dot notation here, but in this particular case I don't think it's useful since there is no `is_vonN_bounded` hypothesis to use dot notation on

Reply:
I have no strong feeling either way.

Reply:
Yeah, I guess I really meant in the few lemmas following this one, which you already mentioned.